integration_test/gce/gce_testing.go

```diff
@@ -21,55 +21,71 @@ To run a test based on this library, you can either:
-    the REPO_SUFFIX for that build and add it as an environment variable to the
-    command below; for example: REPO_SUFFIX=20210805-2. You can also use
-	AGENT_PACKAGES_IN_GCS, for details see README.md.
-
-PROJECT=dev_project \
-    ZONE=us-central1-b \
-    PLATFORMS=debian-10,centos-8,rhel-8-1-sap-ha,sles-15,ubuntu-2004-lts,windows-2012-r2,windows-2019 \
-    go test -v ops_agent_test.go \
-	-test.parallel=1000 \
-	-tags=integration_test \
-    -timeout=4h
+
+Ops Agent code, first build and upload a package to Rapture. Then look up
+the REPO_SUFFIX for that build and add it as an environment variable to the
+command below; for example: REPO_SUFFIX=20210805-2. You can also use
+AGENT_PACKAGES_IN_GCS, for details see README.md.
+
+			PROJECT=dev_project \
+
+	    ZONE=us-central1-b \
+	    PLATFORMS=debian-10,centos-8,rhel-8-1-sap-ha,sles-15,ubuntu-2004-lts,windows-2012-r2,windows-2019 \
+	    go test -v ops_agent_test.go \
+		-test.parallel=1000 \
+		-tags=integration_test \
+	    -timeout=4h
 
 This library needs the following environment variables to be defined:
 
 PROJECT: What GCP project to use.
 ZONE: What GCP zone to run in.
 WINRM_PAR_PATH: (required for Windows) Path to winrm.par, used to connect to
-    Windows VMs.
+
+	Windows VMs.
 
 The following variables are optional:
 
 TEST_UNDECLARED_OUTPUTS_DIR: A path to a directory to write log files into.
-    By default, a new temporary directory is created.
+
+	By default, a new temporary directory is created.
+
 NETWORK_NAME: What GCP network name to use.
 KOKORO_BUILD_ID: supplied by Kokoro.
 KOKORO_BUILD_ARTIFACTS_SUBDIR: supplied by Kokoro.
 LOG_UPLOAD_URL_ROOT: A URL prefix (remember the trailing "/") where the test
-    logs will be uploaded. If unset, this will point to
-    ops-agents-public-buckets-test-logs, which should work for all tests
-    triggered from GitHub.
+
+	logs will be uploaded. If unset, this will point to
+	ops-agents-public-buckets-test-logs, which should work for all tests
+	triggered from GitHub.
+
 USE_INTERNAL_IP: Whether to try to connect to the VMs' internal IP addresses
-    (if set to "true"), or external IP addresses (in all other cases).
-    Only useful on Kokoro.
+
+	(if set to "true"), or external IP addresses (in all other cases).
+	Only useful on Kokoro.
+
 SERVICE_EMAIL: If provided, which service account to use for spawned VMs. The
-    default is the project's "Compute Engine default service account".
+
+	default is the project's "Compute Engine default service account".
+
 TRANSFERS_BUCKET: A GCS bucket name to use to transfer files to testing VMs.
-    The default is "stackdriver-test-143416-file-transfers".
+
+	The default is "stackdriver-test-143416-file-transfers".
+
 INSTANCE_SIZE: What size of VMs to make. Passed in to gcloud as --machine-type.
-    If provided, this value overrides the selection made by the callers to
-		this library.
+
+	    If provided, this value overrides the selection made by the callers to
+			this library.
```

Comment:
```suggestion
If provided, this value overrides the selection made by the callers to
this library.
```

Reply:
Changed in: [41a2e8e](https://github.com/GoogleCloudPlatform/ops-agent/pull/719/commits/41a2e8ed61e2d4c11260b47eebb6a87be026549f)